config/jobs/kubernetes/wg-k8s-infra/k8sio-presubmit.yaml

```diff
@@ -20,6 +20,87 @@ presubmits:
         env:
         - name: GO111MODULE
           value: "on"
+  - name: pull-k8sio-terraform-aaa
+    annotations:
+      description: verify terraform files for GKE cluster aaa
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: wg-k8s-infra-k8sio
+      testgrid-tab-name: pull-k8sio-terraform-aaa
+      testgrid-alert-email: ameukam@gmail.com,spiffxp@gmail.com,spiffxp@google.com
+      testgrid-num-failures-to-alert: '1'
+    decorate: true
+    max_concurrency: 1
+    cluster: k8s-infra-prow-build
+    path_alias: k8s.io/k8s.io
+    run_if_changed: "^(infra/gcp/clusters/modules/.*.tf$|infra/gcp/clusters/projects/kubernetes-public/aaa/*.tf$)"
+    branches:
+    - ^main$
+    spec:
+      containers:
+      - image: docker pull gcr.io/k8s-testimages/gcb-docker-gcloud:v20210331-c732583
+        command:
+        - hack/verify-terraform.sh infra/gcp/clusters/projects/kubernetes-public/aaa
```

Comment:
This should be split into two entries in the list otherwise it will attempt to execute a binary named `hack/verify-terraform.sh infra/gcp/clusters/projects/kubernetes-public/aaa` - e.g:

```
command:
- hack/verify-terraform.sh
- infra/gcp/clusters/projects/kubernetes-public/aaa
```